### PR TITLE
Fx 40 compat

### PR DESCRIPTION
**Pull request by mstriemer**
r? @kmaglione @magopian

### CODE REVIEW

File: validator/testcases/regex.py [hunk 1]
```diff
@@ -363,6 +363,67 @@ def tests(self):
...
+            r"\b([gs]etKeywordForBookmark|getURIForKeyword)\b",
+            "The old keywords API is deprecated.",
+            "The old keywords API is deprecated. You should use "
+            "PlacesUtils.keywords instead. See %s for more information."
+            % MDN_DOC % "Mozilla/Tech/Places/Using_the_Places_keywords_API",
+            log_function=self.err.warning,
+            compat_type="warning")
+
+        yield self.get_test(
+            r"\bfuelIApplication\b",
```
Comment (kmaglione) [inline validator/testcases/regex.py]: This is also accessible via the `Application` global in browser windows, and the `extIApplication` interface.

**Comment by kmaglione** [comment]
You have a failing test, and are missing a couple of patterns for the FUEL test, but otherwise looks good. r+wc

**Comment by mstriemer** [comment]
Failed the test to see the URL and forgot to fix it, oops.

For the extra patterns I'm not sure what those should be. I guess `extIApplication` for one but are you saying there's a global called `Application` that implements the `extIApplication` and `fuelIApplication` interfaces?

**Comment by kmaglione** [comment]
Yeah, the global is just a cached instance of the service. It implements those interfaces.

File: jetpack/addon-sdk [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit e2b154aff5c452d3b4f25bd67513122bd6172319
+Subproject commit 74d7ae8f5496979baf1b7e995b38474dc81d5e70
```
Comment (kmaglione) [inline jetpack/addon-sdk]: Did you mean to include this?

Comment (mstriemer) [inline jetpack/addon-sdk]: I did not.

**Comment by kmaglione** [comment]
r+

**Pull request merged**